sql/trino/TrinoParser.g4

```diff
@@ -492,7 +515,7 @@ primaryExpression
     | ARRAY_ LSQUARE_ (expression (COMMA_ expression)*)? RSQUARE_                                                 # arrayConstructor
     | value = primaryExpression LSQUARE_ index = valueExpression RSQUARE_                                         # subscript
     | identifier                                                                                                  # columnReference
-    | base_ = primaryExpression DOT_ fieldName = identifier                                                       # dereference
+    | base = primaryExpression DOT_ fieldName = identifier                                                        # dereference
```

Comment:
You cannot use "base" because this causes the C# target to fail to build. It is a reserved word in C#. Rename it back to "base_".